awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx

```diff
@@ -58,101 +58,93 @@ const RightActionButtonCell = styled(ActionButtonCell)`
   ${rightStyle}
 `;
 
-class TemplateListItem extends Component {
-  render() {
-    const { i18n, template, isSelected, onSelect } = this.props;
-    const canLaunch = template.summary_fields.user_capabilities.start;
-    const missingResourceIcon =
+function TemplateListItem({ i18n, template, isSelected, onSelect }) {
```

Comment:
Most of this file is different simply because I converted it to a functional component.  The real changes are on lines 63-65.